### PR TITLE
add dz60v2 keyboard support

### PR DESCRIPTION
**Pull request by moyi4681**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by kabalage** [comment]
I would love to help somehow. May anyone fix these configs, or we're waiting for the original authors? Or maybe we should just create a new keyboard like `dztech/dz60v2` with just the default config? Thoughts?

File: keyboards/dz60/v2/info.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+    },
+        "brightness_steps": 10,
+        "hue_steps": 10,
+        "led_count": 16,
+        "pin": "B2",
```
Comment (drashna) [inline keyboards/dz60/v2/info.json:41]:
```suggestion
```

File: keyboards/dz60/v2/info.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        "saturation_steps": 10
+    }
```
Comment (drashna) [inline keyboards/dz60/v2/info.json:43]:
```suggestion
    },
    "ws2812": {
        "pin": "B2"
    }
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/dz60/info.json [hunk 3]
```diff
@@ -1,2106 +1,1981 @@
...
-        "caps_lock": "B2",
-        "on_state": 0
-    },
-    "processor": "atmega32u4",
+    "maintainer": "DZTECH",
     "bootloader": "atmel-dfu",
+    "processor": "atmega32u4",
+    "debounce": 5,
+    "diode_direction": "COL2ROW",
     "community_layouts": ["60_ansi", "60_ansi_arrow_split_bs_7u_spc", "60_ansi_arrow", "60_ansi_split_bs_rshift", "60_hhkb", "60_iso", "60_abnt2", "60_tsangan_hhkb"],
```
Comment (dunk2k) [inline keyboards/dz60/info.json:9]: add 64_* community layouts
```suggestion
    "community_layouts": ["60_ansi", "60_ansi_arrow_split_bs_7u_spc", "60_ansi_arrow", "60_ansi_split_bs_rshift", "60_hhkb", "60_iso", "60_abnt2", "60_tsangan_hhkb", "64_ansi", "64_iso"],
```

File: keyboards/dz60/info.json [hunk 4]
```diff
@@ -1,2106 +1,1981 @@
...
+                { "label": "k41", "matrix": [4, 1], "w": 1.25, "x": 1.25, "y": 4 },
+                { "label": "k43", "matrix": [4, 3], "w": 1.25, "x": 2.5, "y": 4 },
+                { "label": "k46", "matrix": [4, 6], "w": 6.25, "x": 3.75, "y": 4 },
+                { "label": "k4a", "matrix": [4, 10],  "x": 10, "y": 4 },
+                { "label": "k4b", "matrix": [4, 11],  "x": 11, "y": 4 },
+                { "label": "k4c", "matrix": [4, 12],  "x": 12, "y": 4 },
+                { "label": "k4d", "matrix": [4, 13],  "x": 13, "y": 4 },
+                { "label": "k4e", "matrix": [4, 14],  "x": 14, "y": 4 }
             ]
         },
```
Comment (dunk2k) [inline keyboards/dz60/info.json:1705]: add LAYOUT_64_iso layouts
```suggestion
        },
        "LAYOUT_64_iso": {
            "layout": [
                { "label": "k00", "matrix": [0, 0], "x": 0, "y": 0 },
                { "label": "k01", "matrix": [0, 1], "x": 1, "y": 0 },
                { "label": "k02", "matrix": [0, 2], "x": 2, "y": 0 },
                { "label": "k03", "matrix": [0, 3], "x": 3, "y": 0 },
                { "label": "k04", "matrix": [0, 4], "x": 4, "y": 0 },
                { "label": "k05", "matrix": [0, 5], "x": 5, "y": 0 },
                { "label": "k06", "matrix": [0, 6], "x": 6, "y": 0 },
                { "label": "k07", "matrix": [0, 7], "x": 7, "y": 0 },
                { "label": "k08", "matrix": [0, 8], "x": 8, "y": 0 },
                { "label": "k09", "matrix": [0, 9], "x": 9, "y": 0 },
                { "label": "k0a", "matrix": [0, 10], "x": 10, "y": 0 },
                { "label": "k0b", "matrix": [0, 11], "x": 11, "y": 0 },
                { "label": "k0c", "matrix": [0, 12], "x": 12, "y": 0 },
                { "label": "k0e", "matrix": [0, 14], "x": 13, "y": 0, "w": 2 },

                { "label": "k10", "matrix": [1, 0], "x": 0, "y": 1, "w": 1.5 },
                { "label": "k12", "matrix": [1, 2], "x": 1.5, "y": 1 },
                { "label": "k13", "matrix": [1, 3], "x": 2.5, "y": 1 },
                { "label": "k14", "matrix": [1, 4], "x": 3.5, "y": 1 },
                { "label": "k15", "matrix": [1, 5], "x": 4.5, "y": 1 },
                { "label": "k16", "matrix": [1, 6], "x": 5.5, "y": 1 },
                { "label": "k17", "matrix": [1, 7], "x": 6.5, "y": 1 },
                { "label": "k18", "matrix": [1, 8], "x": 7.5, "y": 1 },
                { "label": "k19", "matrix": [1, 9], "x": 8.5, "y": 1 },
                { "label": "k1a", "matrix": [1, 10], "x": 9.5, "y": 1 },
                { "label": "k1b", "matrix": [1, 11], "x": 10.5, "y": 1 },
                { "label": "k1c", "matrix": [1, 12], "x": 11.5, "y": 1 },
                { "label": "k1d", "matrix": [1, 13], "x": 12.5, "y": 1 },

                { "label": "k20", "matrix": [2, 0], "x": 0, "y": 2, "w": 1.75 },
                { "label": "k22", "matrix": [2, 2], "x": 1.75, "y": 2 },
                { "label": "k23", "matrix": [2, 3], "x": 2.75, "y": 2 },
                { "label": "k24", "matrix": [2, 4], "x": 3.75, "y": 2 },
                { "label": "k25", "matrix": [2, 5], "x": 4.75, "y": 2 },
                { "label": "k26", "matrix": [2, 6], "x": 5.75, "y": 2 },
                { "label": "k27", "matrix": [2, 7], "x": 6.75, "y": 2 },
                { "label": "k28", "matrix": [2, 8], "x": 7.75, "y": 2 },
                { "label": "k29", "matrix": [2, 9], "x": 8.75, "y": 2 },
                { "label": "k2a", "matrix": [2, 10], "x": 9.75, "y": 2 },
                { "label": "k2b", "matrix": [2, 11], "x": 10.75, "y": 2 },
                { "label": "k2c", "matrix": [2, 12], "x": 11.75, "y": 2 },
                { "label": "k1e", "matrix": [1, 14], "x": 12.75, "y": 2 },
                { "label": "k2d", "matrix": [2, 13], "x": 13.75, "y": 1, "w":1.25, "h":2 },

                { "label": "k30", "matrix": [3, 0], "x": 0, "y": 3 },
                { "label": "k31", "matrix": [3, 1], "x": 1, "y": 3 },
                { "label": "k32", "matrix": [3, 2], "x": 2, "y": 3 },
                { "label": "k33", "matrix": [3, 3], "x": 3, "y": 3 },
                { "label": "k34", "matrix": [3, 4], "x": 4, "y": 3 },
                { "label": "k35", "matrix": [3, 5], "x": 5, "y": 3 },
                { "label": "k36", "matrix": [3, 6], "x": 6, "y": 3 },
                { "label": "k37", "matrix": [3, 7], "x": 7, "y": 3 },
                { "label": "k38", "matrix": [3, 8], "x": 8, "y": 3 },
                { "label": "k39", "matrix": [3, 9], "x": 9, "y": 3 },
                { "label": "k3a", "matrix": [3, 10], "x": 10, "y": 3 },
                { "label": "k3b", "matrix": [3, 11], "x": 11, "y": 3 },
                { "label": "k3c", "matrix": [3, 12], "x": 12, "y": 3 },
                { "label": "k3d", "matrix": [3, 13], "x": 13, "y": 3 },
                { "label": "k3e", "matrix": [3, 14], "x": 14, "y": 3 },

                { "label": "k40", "matrix": [4, 0], "x": 0, "y": 4, "w": 1.25 },
                { "label": "k41", "matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25 },
                { "label": "k43", "matrix": [4, 3], "x": 2.5, "y": 4, "w": 1.25 },
                { "label": "k46", "matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25 },
                { "label": "k4a", "matrix": [4, 10],  "x": 10, "y": 4 },
                { "label": "k4b", "matrix": [4, 11],  "x": 11, "y": 4 },
                { "label": "k4c", "matrix": [4, 12],  "x": 12, "y": 4 },
                { "label": "k4d", "matrix": [4, 13],  "x": 13, "y": 4 },
                { "label": "k4e", "matrix": [4, 14],  "x": 14, "y": 4 }
            ]
        },
        "LAYOUT_64_iso_split_bs": {
            "layout": [
                { "label": "k00", "matrix": [0, 0], "x": 0, "y": 0 },
                { "label": "k01", "matrix": [0, 1], "x": 1, "y": 0 },
                { "label": "k02", "matrix": [0, 2], "x": 2, "y": 0 },
                { "label": "k03", "matrix": [0, 3], "x": 3, "y": 0 },
                { "label": "k04", "matrix": [0, 4], "x": 4, "y": 0 },
                { "label": "k05", "matrix": [0, 5], "x": 5, "y": 0 },
                { "label": "k06", "matrix": [0, 6], "x": 6, "y": 0 },
                { "label": "k07", "matrix": [0, 7], "x": 7, "y": 0 },
                { "label": "k08", "matrix": [0, 8], "x": 8, "y": 0 },
                { "label": "k09", "matrix": [0, 9], "x": 9, "y": 0 },
                { "label": "k0a", "matrix": [0, 10], "x": 10, "y": 0 },
                { "label": "k0b", "matrix": [0, 11], "x": 11, "y": 0 },
                { "label": "k0c", "matrix": [0, 12], "x": 12, "y": 0 },
                { "label": "k0d", "matrix": [0, 13], "x": 13, "y": 0 },
                { "label": "k0e", "matrix": [0, 14], "x": 14, "y": 0 },

                { "label": "k10", "matrix": [1, 0], "x": 0, "y": 1, "w": 1.5 },
                { "label": "k12", "matrix": [1, 2], "x": 1.5, "y": 1 },
                { "label": "k13", "matrix": [1, 3], "x": 2.5, "y": 1 },
                { "label": "k14", "matrix": [1, 4], "x": 3.5, "y": 1 },
                { "label": "k15", "matrix": [1, 5], "x": 4.5, "y": 1 },
                { "label": "k16", "matrix": [1, 6], "x": 5.5, "y": 1 },
                { "label": "k17", "matrix": [1, 7], "x": 6.5, "y": 1 },
                { "label": "k18", "matrix": [1, 8], "x": 7.5, "y": 1 },
                { "label": "k19", "matrix": [1, 9], "x": 8.5, "y": 1 },
                { "label": "k1a", "matrix": [1, 10], "x": 9.5, "y": 1 },
                { "label": "k1b", "matrix": [1, 11], "x": 10.5, "y": 1 },
                { "label": "k1c", "matrix": [1, 12], "x": 11.5, "y": 1 },
                { "label": "k1d", "matrix": [1, 13], "x": 12.5, "y": 1 },

                { "label": "k20", "matrix": [2, 0], "x": 0, "y": 2, "w": 1.75 },
                { "label": "k22", "matrix": [2, 2], "x": 1.75, "y": 2 },
                { "label": "k23", "matrix": [2, 3], "x": 2.75, "y": 2 },
                { "label": "k24", "matrix": [2, 4], "x": 3.75, "y": 2 },
                { "label": "k25", "matrix": [2, 5], "x": 4.75, "y": 2 },
                { "label": "k26", "matrix": [2, 6], "x": 5.75, "y": 2 },
                { "label": "k27", "matrix": [2, 7], "x": 6.75, "y": 2 },
                { "label": "k28", "matrix": [2, 8], "x": 7.75, "y": 2 },
                { "label": "k29", "matrix": [2, 9], "x": 8.75, "y": 2 },
                { "label": "k2a", "matrix": [2, 10], "x": 9.75, "y": 2 },
                { "label": "k2b", "matrix": [2, 11], "x": 10.75, "y": 2 },
                { "label": "k2c", "matrix": [2, 12], "x": 11.75, "y": 2 },
                { "label": "k1e", "matrix": [1, 14], "x": 12.75, "y": 2 },
                { "label": "k2d", "matrix": [2, 13], "x": 13.75, "y": 1, "w":1.25, "h":2 },

                { "label": "k30", "matrix": [3, 0], "x": 0, "y": 3 },
                { "label": "k31", "matrix": [3, 1], "x": 1, "y": 3 },
                { "label": "k32", "matrix": [3, 2], "x": 2, "y": 3 },
                { "label": "k33", "matrix": [3, 3], "x": 3, "y": 3 },
                { "label": "k34", "matrix": [3, 4], "x": 4, "y": 3 },
                { "label": "k35", "matrix": [3, 5], "x": 5, "y": 3 },
                { "label": "k36", "matrix": [3, 6], "x": 6, "y": 3 },
                { "label": "k37", "matrix": [3, 7], "x": 7, "y": 3 },
                { "label": "k38", "matrix": [3, 8], "x": 8, "y": 3 },
                { "label": "k39", "matrix": [3, 9], "x": 9, "y": 3 },
                { "label": "k3a", "matrix": [3, 10], "x": 10, "y": 3 },
                { "label": "k3b", "matrix": [3, 11], "x": 11, "y": 3 },
                { "label": "k3c", "matrix": [3, 12], "x": 12, "y": 3 },
                { "label": "k3d", "matrix": [3, 13], "x": 13, "y": 3 },
                { "label": "k3e", "matrix": [3, 14], "x": 14, "y": 3 },

                { "label": "k40", "matrix": [4, 0], "x": 0, "y": 4, "w": 1.25 },
                { "label": "k41", "matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25 },
                { "label": "k43", "matrix": [4, 3], "x": 2.5, "y": 4, "w": 1.25 },
                { "label": "k46", "matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25 },
                { "label": "k4a", "matrix": [4, 10],  "x": 10, "y": 4 },
                { "label": "k4b", "matrix": [4, 11],  "x": 11, "y": 4 },
                { "label": "k4c", "matrix": [4, 12],  "x": 12, "y": 4 },
                { "label": "k4d", "matrix": [4, 13],  "x": 13, "y": 4 },
                { "label": "k4e", "matrix": [4, 14],  "x": 14, "y": 4 }
            ]
        },
```

File: keyboards/dz60/keymaps/via/keymap.c
Comment (dunk2k) [inline keyboards/dz60/keymaps/via/keymap.c:1]: could this keymap's layout be changed to the `LAYOUT_all` layout (in `info.json`)

File: keyboards/dz60/keymaps/default/keymap.c
Comment (dunk2k) [inline keyboards/dz60/keymaps/default/keymap.c:1]: could this keymap's layout be changed to the `LAYOUT_all` layout (in `info.json`)

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Pull request closed without merging**